visualization/app/codeCharta/util/globalSettingsHelper.ts

```diff
@@ -0,0 +1,44 @@
+"use strict"
+import {GlobalSettings, LocalStorageGlobalSettings } from "../codeCharta.model"
+import { StoreService } from "../state/store.service"
+import { setExperimentalFeaturesEnabled } from "../state/store/appSettings/enableExperimentalFeatures/experimentalFeaturesEnabled.actions"
+import { setHideFlatBuildings } from "../state/store/appSettings/hideFlatBuildings/hideFlatBuildings.actions"
+import { setIsWhiteBackground } from "../state/store/appSettings/isWhiteBackground/isWhiteBackground.actions"
+import { setLayoutAlgorithm } from "../state/store/appSettings/layoutAlgorithm/layoutAlgorithm.actions"
+import { setMaxTreeMapFiles } from "../state/store/appSettings/maxTreeMapFiles/maxTreeMapFiles.actions"
+import { setResetCameraIfNewFileIsLoaded } from "../state/store/appSettings/resetCameraIfNewFileIsLoaded/resetCameraIfNewFileIsLoaded.actions"
+
+
+export class GlobalSettingsHelper {
+    static readonly GLOBALSETTINGS_LOCAL_STORAGE_VERSION = "1.0.0"
+	static readonly GLOBALSETTINGS_LOCAL_STORAGE_ELEMENT = "globalSettings"
+	
+	static setGlobalSettingsInLocalStorage(globalSettings: GlobalSettings){
+            const newLocalStorageElement: LocalStorageGlobalSettings = {
+                version: this.GLOBALSETTINGS_LOCAL_STORAGE_VERSION,
```

Comment:
Let's the actual visualization version instead without using it for the lookup. If we ever encounter new entries, it's possible to add those without any negative impact. If we remove something, it's also fine, since we just won't use that entry anymore. If we change something (which we likely shouldn't) it's possible to handle the difference between versions while implementing the changed behavior.